src/main/java/edu/cornell/kfs/concur/service/ConcurCashAdvancePdpFeedFileService.java

```diff
@@ -0,0 +1,9 @@
+package edu.cornell.kfs.concur.service;
+
+import org.kuali.kfs.sys.batch.InitiateDirectory;
+
+public interface ConcurCashAdvancePdpFeedFileService extends InitiateDirectory {
+
+	public void createDoneFileFor(String concurCashAdvancePdpFeedFileName);
```

Comment:
public modifier can be removed from these methods as they are public by default.